dist/tools/ci/can_fast_ci_run.py

```diff
@@ -164,14 +164,37 @@ def classify_changes(riotbase=None, upstream_branch="master"):
             match = REGEX_GIT_DIFF_SINGLE_FILE.match(line)
             if match:
                 file = match.group(1)
-                change_set.add_file(file)
```

Comment:
Seeing the use of the builtin`file` as a variable and I think it should be renamed as filename. I know it's already in master but might be worth fix it.

Reply:
follow-up?

Reply:
Not needed, I double checked and in Python `file` is no longer a builtin. So that's fine to use it.

Reply:
Thanks for checking :)